Quotas/README.md

```diff
@@ -0,0 +1,45 @@
+# Google Cloud Cloud Quotas for PHP
+
+> Idiomatic PHP client for [Google Cloud Cloud Quotas](https://cloud.google.com).
+
+[![Latest Stable Version](https://poser.pugx.org/google/api-cloudquotas/v/stable)](https://packagist.org/packages/google/api-cloudquotas) [![Packagist](https://img.shields.io/packagist/dm/google/api-cloudquotas.svg)](https://packagist.org/packages/google/api-cloudquotas)
+
+* [API documentation](https://cloud.google.com/php/docs/reference/api-cloudquotas/latest)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googleapis/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now, install this component:
+
+```sh
+$ composer require google/api-cloudquotas
```

Comment:
Shouldn't this be `google/cloud-quotas`? 

The suggestion based on the proto directory is just a suggestion. I don't think it makes sense to call it `api-cloudquotas`, we don't have any other packages with this precedent